Broken postinstall script in stonyx@0.2.3-beta.44 breaks `pnpm install`

## Bug

`stonyx@0.2.3-beta.44` has a `postinstall` script declared in `package.json` that points to `scripts/postinstall.js`, but the published tarball does not include the `scripts/` directory. Every consumer that runs `pnpm install` / `npm install` fails with:

```
node:internal/modules/cjs/loader:1386
  throw err;
  ^
Error: Cannot find module '/home/runner/work/<repo>/<repo>/node_modules/.pnpm/stonyx@0.2.3-beta.44/node_modules/stonyx/scripts/postinstall.js'
    at Function._resolveFilename (node:internal/modules/cjs/loader:1383:15)
    ...
  code: 'MODULE_NOT_FOUND',
}
 ELIFECYCLE  Command failed with exit code 1.
```

This breaks CI for **every downstream Stonyx consumer** — confirmed in `abofs/nextgoal-alerts` where it is blocking Sprint 35 Wave 1 merge (PR https://github.com/abofs/nextgoal-alerts/pull/48).

## Root cause (likely)

Either:
- `scripts/postinstall.js` was added to the repo but is excluded from `files` in `package.json` so it never makes it into the published tarball, OR
- The `postinstall` hook was added before the script file, OR
- `.npmignore` / `files` glob is stripping it.

## Expected

`pnpm install` / `npm install` of any version of `stonyx` succeeds cleanly. No postinstall lifecycle errors.

## Impact

**P1** — blocks all downstream Stonyx consumers from `pnpm install` in clean environments (CI, fresh clones). NextGoal Sprint 35 PR https://github.com/abofs/nextgoal-alerts/pull/48 cannot merge until this is resolved.

## Suggested fix

1. Either publish a patch version with the `scripts/postinstall.js` file included in the tarball, OR
2. Remove the `postinstall` script from `package.json` if it's no longer needed, OR
3. Make the postinstall self-heal when the script is absent.

## Verification

After publishing a fix, re-run CI on nextgoal-alerts PR #48 — `pnpm install --frozen-lockfile` should succeed.

--- a/package.json
+++ b/package.json
@@ -46,6 +46,7 @@
   "files": [
     "dist",
     "config",
+    "scripts",
     "README.md"
   ],
   "publishConfig": {

